docs/beginners-guide-to-dml-and-pydml.md

```diff
@@ -36,10 +36,8 @@ that comes in two flavors, one with an R-like syntax (DML) and one with
 a Python-like syntax (PyDML).
 
 Algorithm scripts written in DML and PyDML can be run on Hadoop, on Spark, or
-in Standalone mode. No script modifications are required to change between modes.
-SystemML automatically performs advanced
-optimizations based on data and cluster characteristics, so much of the need to manually
-tweak algorithms is largely reduced or eliminated.
+in Standalone mode. SystemML also features an MLContext API that allows SystemML
+to be accessed from a Spark Shell, a Jupyter Notebook, or a Zeppelin Notebook.
```

Comment:
maybe note that it can be used from Scala & Python in this sentence?  `... accessed via Scala or Python from a Spark Shell...`